src/schema/v2/artist/targetSupply/index.ts

```diff
@@ -11,8 +11,8 @@ import { ResolverContext } from "types/graphql"
 import { getArtistMicrofunnelMetadata } from "./utils/getMicrofunnelData"
 
 const ARTIST_TARGET_SUPPLY_PRIORITIES = {
-  P1: { value: 1 },
-  P2: { value: 2 },
+  FALSE: { value: 0 },
+  TRUE: { value: 1 },
```

Comment:
This is a specific request from the Data team for these values to remain integers but to be used as booleans.